dev-tools/composer.json

```diff
@@ -5,7 +5,6 @@
         "icanhazstring/composer-unused": "^0.8.11",
         "maglnet/composer-require-checker": "^4.7.1",
         "mi-schi/phpmd-extension": "^4.3.0",
-        "pdepend/pdepend": "~2.15.0",
```

Comment:
can we somehow prevent installing wrong version?
eg require version of direct dependencies that enforce high-level peer-dependencies, or with conflicts?

i imagine someone installing with --prefer-lowest etc, and getting broken variant of this lib

Reply:
The other repo should do that, not us. It's wrong for us to intrude in that way, or even assume they will keep using that lib at all. The risk is low anyway IMO, since the intended use case is to install from lock file.